js/views/pageNavVw.js

```diff
@@ -33,7 +33,7 @@ module.exports = baseVw.extend({
     'click .js-aboutModal .js-tab': 'aboutModalTabClick',
     'click .js-navRefresh': 'navRefreshClick',
     'click .js-navAdminPanel': 'navAdminPanel',
-    'click .js-navProfileMenu a': 'closeNav',
```

Comment:
You changed the name of the function in the events hash, but you didn't actually change the function name, so the handler will never get called.